src/targets/services/onOperationOrBillCreate.js

```diff
@@ -158,7 +158,12 @@ const onOperationOrBillCreate = async options => {
   setting = await updateSettings(setting)
 
   log('info', 'Do category budgets notification')
-  await runCategoryBudgetService(cozyClient)
+  const client = CozyClient.fromEnv(process.env)
+  const jobs = client.collection('io.cozy.jobs')
+  await jobs.create('service', {
+    name: 'budgetAlerts',
+    slug: 'banks'
+  })
```

Comment:
Out of curiosity, why the change from service to job?

Reply:
"from function to job" I guess. Because this enables finer grained monitoring in Bender and in metabase.